third_party/libjpeg-turbo/turbojpeg.c

```diff
@@ -44,17 +44,13 @@
 #include "./cdjpeg.h"
 
 #ifdef STARBOARD
-#include "starboard/client_porting/poem/strings_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
+#include "starboard/common/log.h"
```

Comment:
I see this was included indirectly through `string_poem.h`.
Is it actually needed?

Reply:
It provides `SB_DCHECK`, used a few places below.